ydkgen/resolver/bundle_resolver.py

```diff
@@ -81,8 +81,8 @@ class BundleDefinition(object):
 
     def __init__(self, data):
         self._name = data['name'].replace('-', '_')
-        self._version = Version(*tuple(data['version'].split('.')))
-        self._core_version = Version(*tuple(data['core-version'].split('.')))
+        self._version = Version(data['version'])
```

Comment:
This can potentially break line 101 because of the type change

Reply:
It looks like line 101 is only used by bundle_translator.py to render TEMPLATE. That should end up calling str() on the property.

Reply:
It seems the proposed change adds a new dependency.  Would like to explore an alternative that doesn't introduce one if possible.

Reply:
Packaging dependency is using this big regex to match PEP440: https://github.com/pypa/packaging/blob/2a883ca86300bab76ad6296860797907b3432871/packaging/version.py#L249
I guess the Version class can be copied and pasted as is if you want to avoid a dependency, but I'd rather prefer a new dependency rather than a copying and pasting code.